.dockerignore

```diff
@@ -1,15 +1,64 @@
+# vim:set ft=gitignore:
+
+# Git / Editor / Coverage / Linter
+.git
+.gitignore
+.dockerignore
+.idea
+*.swo
+*.swp
+html/
+htmlcov/
+.coverage
+.coverage.*
+.coveragerc
+coverage.xml
+*,cover
+cover/
+.pytest_cache/
+.tox/
+.cache
+nosetests.xml
+.mypy_cache/
+
+# CI
+.appveyor.yml
+.bumpversion.cfg
+.circleci
+.pyup.yml
+
+# Synapse
+docs/
+CHANGELOG.md
+Dockerfile
+MANIFEST.in
```

Comment:
Does the `python setup.py install` work properly without the MANIFEST.in being present?

Reply:
Ya. I'll add it back anyway.